spacy/tests/lang/nl/test_noun_chunks.py

```diff
@@ -207,3 +208,18 @@ def test_chunking(nl_sample, nl_reference_chunking):
     """
     chunks = [s.text.lower() for s in nl_sample.noun_chunks]
     assert chunks == nl_reference_chunking
+
+
+@pytest.mark.issue(10846)
+def test_no_overlapping_chunks(nl_vocab):
+    # fmt: off
```

Comment:
What do you mean with fmt?

Reply:
"fmt" is short for "formatting" - it's a meta code signaling to the `black` formatting tool that the following lines can be kept as-is.